packages/rx_bloc_cli/bin/compile.sh

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env sh
-flutter pub run mason bundle -t dart mason_templates/bricks/rx_bloc_base
+dart pub run mason bundle -t dart mason_templates/bricks/rx_bloc_base
 cp rx_bloc_base_bundle.dart lib/src/templates/
 rm rx_bloc_base_bundle.dart
-pub global activate -s path . --overwrite
-rx_bloc_cli create --org com.primeholding --project-name test_app --include-analytics true example/test_app
+dart pub global activate -s path . --overwrite
+rm -rf example/test_app
+rx_bloc_cli create --org com.primeholding --project-name ccvoiptest --include-analytics true example/test_app
+#rx_bloc_cli create --org com.primeholding --project-name test_app --include-analytics true example/test_app
```

Comment:
When a project is generated, it includes push notifications preconfigured. The only thing the developer needs to take care is to register the app inside Firebase and download configuration files in order to use the push notifications. 

Since on iOS, we need to test push notifications on a real device (and in order to install it, we need provisioning profiles for an app with a name in a specific format) we used a previous test project for the same purpose. This way, you'd only need to generate the project with the specific name and drop those config files, so we're good to go testing the app.

Reply:
Thank you for this detail explanation!